core.yaml

```diff
@@ -389,7 +389,11 @@ security:
     # Can be 256 or 384. If you change here, you have to change also
     # the same property in membersrvc.yaml to the same value
     level: 256
-
+ 
+    # Can be SHA2 or SHA3. If you change here, you have to change also
+    # the same property in membersrvc.yaml to the same value
+    hashAlgorithm: SHA3
```

Comment:
@srderson, I think that it's better make the util method take these properties as parameters and then define properties separately for different usages (e.g., one we can define for state).